index.bs

```diff
@@ -2449,6 +2442,13 @@ To <dfn>obtain and deliver a debug report on source registration</dfn> given a
     : "<code>[=source debug data type/source-storage-limit=]</code>"
     :: [=map/Set=] |body|["`limit`"] to the user agent's [=max pending sources per source origin=],
          [=serialize an integer|serialized=].
+    : "<code>[=source debug data type/source-channel-capacity-limit=]</code>"
+    ::
+        1. Let |sourceType| be |source|'s [=attribution source/source type=].
+        1. [=map/Set=] |body|["`limit`"] to the user agent's [=max event-level channel capacity per source=][|sourceType|].
```

Comment:
Does it matter that this will be the first debug report whose `limit` field contains a JSON number rather than a string?

Reply:
I thought about that as well, and feel that it’s more aligned with other JSON number values in the registration JSON or event-level report.